man/common/tcti.md

```diff
@@ -1,90 +1,106 @@
+    Example: **-T device:/dev/tpm0** or
+    **export _TPM2TOOLS\_TCTI\_NAME_="device:/dev/tpm0"**
 
-    * **mssim**: For the mssim TCTI, the domain name or IP address and port number used by the simulator
-      can be specified. The default are 127.0.0.1 and 2321.
-      Example: **-T mssim:tcp://127.0.0.1:2321** or **export _TPM2TOOLS\_TCTI\_NAME_="mssim:tcp://127.0.0.1:2321"**
+  * **mssim**:
+  * For the mssim TCTI, the domain name or IP address and port number used by
+    the simulator can be specified. The default are 127.0.0.1 and 2321.
 
-    * **abrmd**: For the abrmd TCTI, the configuration string format is a series of simple key value pairs
-      separated by a ',' character. Each key and value string are separated by a '=' character.
+    Example: **-T mssim:tcp://127.0.0.1:2321** or
+    **export _TPM2TOOLS\_TCTI\_NAME_="mssim:tcp://127.0.0.1:2321"**
 
-        * TCTI abrmd supports two keys:
-            1. 'bus_name' : The name of the tabrmd service on the bus (a string).
-            2. 'bus_type' : The type of the dbus instance (a string) limited to
+  * **abrmd**:
+    For the abrmd TCTI, the configuration string format is a series of simple
+    key value pairs separated by a ',' character. Each key and value string
+    are separated by a '=' character.
+
+      * TCTI abrmd supports two keys:
+
+          1. 'bus_name' : The name of the tabrmd service on the bus (a string).
+          2. 'bus_type' : The type of the dbus instance (a string) limited to
                'session' and 'system'.
 
-        Specify the tabrmd tcti name and a config string of ```bus_name=com.example.FooBar```:
-        ```
-        --tcti=tabrmd:bus_name=com.example.FooBar
-        ```
+      Specify the tabrmd tcti name and a config string of
+      ```bus_name=com.example.FooBar```:
 
-        Specify the default (abrmd) tcti and a config string of ```bus_type=session```:
-        ```
-        --tcti:bus_type=session
-        ```
+      ```
+      --tcti=tabrmd:bus_name=com.example.FooBar
+      ```
 
-        Specify the device tcti and use the default config:
-        ```
-        --tcti=device
-        ```
-        or
-        ```
-        --tcti=device:
-        ```
+      Specify the default (abrmd) tcti and a config string of
+      ```bus_type=session```:
 
-        **NOTE**: abrmd and tabrmd are synonymous.
+      ```
+      --tcti:bus_type=session
+      ```
 
-## TCTI Defaults
-When a TCTI is not specified, the default TCTI is searched for using dlopen(3) semantics.
-The tools will search for *tabrmd*, *device* and *mssim* TCTIs **IN THAT ORDER** and
-**USE THE FIRST ONE FOUND**. You can query what TCTI will be chosen as the default by
-using the `-v` option to print the version information. The "default-tcti" key-value pair
-will indicate which of the aforementioned TCTIs is the default.
+      Specify the device tcti and use the default config:
```

Comment:
Having a device related example under the **abrmd** section confused me, should this be moved up to the **device** section? Or perhaps the goal is to have a section of worked examples at the end of the TCTI section? If so we should add a separator/header for the section.

Reply:
No that should be local to device.